ZeeHomeScreen/Classes/Components/Cell/CellViewController.swift

```diff
@@ -211,14 +211,14 @@ import ApplicasterSDK
         }
     }
     
-    func customizeBackground() {
-        let componentCustomization = ComponentModelCustomization()
-        if let color: UIColor = componentCustomization.color(forAttributeKey: kAttributeBackgroundColorKey, attributesDict: nil, withModel: componentDataSourceModel, componentState: componentState()) {
-            view.backgroundColor = color
-        } else {
-            view.backgroundColor = .black
-        }
-    }
+//    func customizeBackground() {
+//        let componentCustomization = ComponentModelCustomization()
+//        if let color: UIColor = componentCustomization.color(forAttributeKey: kAttributeBackgroundColorKey, attributesDict: nil, withModel: componentDataSourceModel, componentState: componentState()) {
```

Comment:
why did you comment out the use of kAttributeBackgroundColorKey? whouldn't we need it to be able to change bg color from the ui builder?
is it possible to set a clear color in the configurations instead of hardcode it?

Reply:
I try to find how to set a clear color in the configurations but find only by hardcode it
cause it can't be gradient without hardcode to clear, only full color

Reply:
got it, thanks